src/streamlink/plugins/tf1.py

```diff
@@ -42,16 +44,30 @@ def get_stream_urls(self, channel):
     def _get_streams(self):
         m = self.match
         if m:
-            channel = m.group(1) or m.group(2)
+            channel = ""
+            channel_id = ""
+            if m.group("live"):
+                channel = m.group("live")
+                channel_id = "L_" + channel.upper()
+            elif m.group("lci"):
+                channel = "LCI"
+                channel_id = "L_LCI"
+            elif m.group("stream"):
+                channel = m.group("stream")
+                channel_id = "L_FAST_v2l-" + m.group("stream")
```

Comment:
Please remove the unnecessary `if m:` block.

Also please don't use the `+` operator for string concatenation, use f-strings.